packages/forms/src/components/Submit/Submit.tsx

```diff
@@ -41,11 +41,11 @@ const Submit = ({
 
   return (
     <Button
-      color={color || "secondary"}
-      mode={mode || "solid"}
+      color={color ?? "secondary"}
+      mode={mode ?? "solid"}
       type="submit"
       icon={icon}
-      disabled={disabled || isSubmitting || overrideSubmitting}
+      disabled={disabled ?? isSubmitting ?? overrideSubmitting}
```

Comment:
I think theses should be ||.

Reply:
Updated in [ignore boolean primitives](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11651/commits/ccb9ba3ec73672fea328bb97fecdb8b1c1db5b91)